code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -46,6 +46,13 @@
 	for(var/t in xenos_by_tier)
 		. += length(xenos_by_tier[t])
 
+//Normal hive status includes burrowed
+/datum/hive_status/normal/get_total_xeno_number()
```

Comment:
This proc is used elsewhere, so what you want to redefine is `can_hive_have_a_queen()`